lightwood/data/cleaner.py

```diff
@@ -63,6 +63,8 @@ def _standardize_array(element):
         # Handles cases where arrays are numbers
         if ' ' not in element:
             element = _clean_float_or_none(element)
+        else:
+            element = [float(x) for x in element.split(' ')]
```

Comment:
Are you indicated sequences that look like this? "1 2 3 5 6 1 5 5" which is why you split? What happens if a user has them comma separated or something else?

Reply:
Also what if they happen to be integers as opposed to floats?

Reply:
See above lines, this case is handled, arrays are standardized to a ` ` split string and then split... though this might be a bad way of doing it, since it was done when our array columns were typed as `str` in the dataframe, now they use `list`.